content/SCALE/SCALETutorials/SystemSettings/UpdateSCALE.md

```diff
@@ -1,44 +1,49 @@
 ---
-title: "Update"
-geekdocCollapseSection: true
+title: "Updating SCALE"
+description: ""
 weight: 10
+alias: /scale/scaleuireference/systemsettings/updatescale/
+tags:
+ - scaleupdate
 ---
 
 {{< toc >}}
 
 TrueNAS has several software branches (linear update paths) known as trains. SCALE is currently a Prerelease Train. Prerelease Trains have various preview/early build releases of the software. 
 
-SCALE has several trains available for updates. However, the web interface only displays trains you can select as an upgrade. For more information on other available trains, see [Truenas Upgrades](https://www.truenas.com/docs/truenasupgrades/).
+SCALE has several trains available for updates. However, the web interface only displays trains you can select as an upgrade. For more information on other available trains, see [TrueNAS Upgrades](https://www.truenas.com/docs/truenasupgrades/).
 
 {{< hint danger >}}
-**TrueNAS SCALE train is not suited for production use. Only use SCALE for testing.**
+TrueNAS SCALE train is *not* suited for production use. Only use SCALE for testing.
```

Comment:
I need to adjust this statement to indicate which page to check for usage recommendations.